appmonitoring/ts/src/Constants.ts

```diff
@@ -0,0 +1,4 @@
+export const WebhookDNSEndpoint = 'app-monitoring-webhook-service.kube-system.svc';
```

Comment:
Let's make a comment explaining where else this lives - in case we need to change it, or even to warn people against changing it.

Reply:
added the comment in Constants.ts file